### PR TITLE
Fix typo in why3/README

### PR DESCRIPTION
**Pull request by Lysxia**
:smile:

### CODE REVIEW

File: why3/README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Why3 
 
 A Rust library furnishing the Why3 AST used by the Creusot verifier, and its associated pretty printer. 
-The library supports three input formats to why3: WhyML (`.mlw`), coma (`.coma`) and Coma (`.coma`).
+The library supports three input formats to why3: WhyML (`.mlw`), Coma (`.coma`).
```
Comment (Armael) [inline why3/README.md:4]: "three" -> "two"?

Comment (Lysxia) [reply why3/README.md]: Good catch!

**Review by xldenis — APPROVED** [review]
I actually want to remove the mlcfg stuff since I could take advantage of that for some other serious cleanups and improvements.

File: why3/README.md (hunk 1 above)
Comment (xldenis) [reply why3/README.md]: technically it _is_ 3 since its `mlcfg`, `coma` and `mlw`.

**Pull request merged**